config/jobs/kubernetes-sigs/kustomize/kustomize-presubmit-master.yaml

```diff
@@ -15,7 +16,11 @@ presubmits:
         - prow-presubmit-check
         resources:
           requests:
+            memory: "2000Mi"
             cpu: 4000m
+          limits:
+            memory: "2000Mi"
+            cpu: 500m                    
```

Comment:
This will need to be 4000m to match the request

Reply:
I am doing this change. Let's look into the results